assets/cubyz/biomes/prairie/hill.zig.zon

```diff
@@ -0,0 +1,22 @@
+.{
+	.minHeight = 54,
+	.maxHeight = 58,
+	.minRadius = 16,
+	.maxRadius = 80,
```

Comment:
I don't think we should make them this big. I just tested it in-game and came across a rather large continuous hill area.